### PR TITLE
adding ecoregion plot

### PR DESCRIPTION
**Pull request by jread-usgs**
@aappling-usgs and/or @lindsaycarr can you take a look?

### CODE REVIEW

File: gap_analysis/powstream_gap_analysis.Rmd [hunk 1]
```diff
@@ -429,3 +428,98 @@ kable(var_table, caption="Table 1. EflowStats variable definitions")
...
+destination = tempfile(pattern = 'eco_shape', fileext='.zip')
+query <- 'http://cida.usgs.gov/gdp/geoserver/wfs?service=WFS&request=GetFeature&typeName=derivative:Level_III_Ecoregions&outputFormat=shape-zip&version=1.0.0'
+file <- GET(query, write_disk(destination, overwrite=T), progress())
+shp.path <- tempdir()
+unzip(destination, exdir = shp.path)
+eco = readOGR(shp.path, layer='Level_III_Ecoregions') %>% 
+  spTransform(CRS(plot.CRS))
+
+# sites: 
+metabDataAll <- get_meta()
```
Comment (lindsaycarr) [inline gap_analysis/powstream_gap_analysis.Rmd]: This is the same as L23!

Comment (jread-usgs) [inline gap_analysis/powstream_gap_analysis.Rmd]: can yank that. I wasn't able to get your's to run, so I made it independent of the variables you pull in. But yes, I did copy your code here :)

Comment (lindsaycarr) [inline gap_analysis/powstream_gap_analysis.Rmd]: oh yes, we still need to figure out where to store those SB files (someone mentioned using sbtools to get that data, but we tabled it)

**Comment by aappling-usgs** [comment]
looks good to me. are those files pretty fast to download?

yeah, @lindsaycarr , sbtools should be the way to go for mike's data - maybe we can make headway on that bit (should be just a few lines, really) next time we meet.

**Comment by jread-usgs** [comment]
yep, these are relatively fast to run

**Comment by aappling-usgs** [comment]
:+1:

**Pull request merged**